.github/workflows/cronJob.yaml

```diff
@@ -11,29 +11,64 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       pr_details: ${{ steps.extract.outputs.pr_details }}
+      is_empty: ${{ steps.extract.outputs.is_empty }}
     env:
       API_URL: https://api.github.com/repos/redhat-developer/lsp4ij/pulls
+    name: PR Details
     steps:
       - name: Extract PR numbers and merge_commit_shas
         shell: bash
         id: extract
         run: |
-          pr_infos=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
-          "${{ env.API_URL }}")
-
+          # Fetch PR details from the GitHub API
+          pr_infos=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" "${{ env.API_URL }}")
+          
           # Extract PR numbers and merge_commit_sha values, excluding draft pull requests
-          pr_details=$(echo "$pr_infos" | jq -r '.[] | select(.draft == false) | {number: .number, sha: .merge_commit_sha}')
-
-          # Print the PR number and corresponding merge commit sha
-          echo "PR number and merge commit sha:"
-          echo "$pr_details" | jq -r '. | "PR #\(.number): \(.sha)"'
-
-          # Create a JSON array string of PR numbers and SHAs
-          pr_details=$(echo "$pr_details" | jq -nc '[inputs | {number: .number, sha: .sha}]')
-          echo "pr_details=$pr_details" >> $GITHUB_OUTPUT
+          pr_numbers=$(echo "$pr_infos" | jq -r '.[] | select(.draft == false) | {number: .number, sha: .merge_commit_sha} | @base64')          
```

Comment:
@anusreelakshmi934 , Is there any advantage adding '@base64' ?

Reply:
This encodes selected JSON objects into a base64-encoded format, simplifying the handling of JSON data to prevent issues when passing data between different sections of a script.